R/calculateUnifrac.R

```diff
@@ -121,19 +125,19 @@ setMethod("calculateUnifrac",
         signature = c(x = "TreeSummarizedExperiment",
                     tree = "missing"),
     function(x, assay.type = assay_name, assay_name = exprs_values, exprs_values = "counts", 
-            tree_name = "phylo", transposed = FALSE, ...){
+             tree.name = tree_name, tree_name = "phylo", transposed = FALSE, ...){
```

Comment:
INdentation seems to be off, run BiocCheck::BiocCheck() and check every line that is modified in this PR

Reply:
I think you can manually revert these lines. There are not too many